index.html

```diff
@@ -308,9 +308,24 @@ <h2>
           Permission
         </h2>
         <p>
-          The term <dfn>express permission</dfn> refers to an act by the user, e.g. via user
-          interface or host device platform features, via which the user approves the use of the
-          Push API by the web application.
+          The Push API is a [=powerful feature=], requiring [=express permission=] to be used.
+        </p>
+        <p>
+          For integration with the [[[Permissions]]] specification, this specification defines the
+          {{PushPermissionDescriptor}} [=powerful feature/permission descriptor type=].
+        </p>
+        <pre class="idl">
+          dictionary PushPermissionDescriptor : PermissionDescriptor {
+            boolean userVisibleOnly = false;
+          };
+        </pre>
+        <p data-dfn-for="PushPermissionDescriptor">
+          The <dfn>userVisibleOnly</dfn> has the same semantics as
```

Comment:
```suggestion
          The <dfn>userVisibleOnly</dfn> member has the same semantics as
```

Reply:
```suggestion
          <dfn>userVisibleOnly</dfn> has the same semantics as
```
I think that you need to drop the article after your tweak.